src/components/ContainerSettingsLinks.react.js

```diff
@@ -0,0 +1,162 @@
+var _ = require('underscore');
+var React = require('react/addons');
+var remote = require('remote');
+var metrics = require('../utils/MetricsUtil');
+var dialog = remote.require('dialog');
+var ContainerUtil = require('../utils/ContainerUtil');
+var containerActions = require('../actions/ContainerActions');
+var containerStore = require('../stores/ContainerStore');
+var util = require('../utils/Util');
+var Typeahead = require('react-typeahead').Typeahead;
+
+var ContainerSettingsLinks = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    let links = ContainerUtil.links(this.props.container) || [];
+    links.push(['', '']);
```

Comment:
For a future PR:  maybe we can create a common 'list' component for env vars, links, basically any tables.

cc @Elesant